ci-automation/vendor-testing/qemu.sh

```diff
@@ -3,66 +3,48 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
-set -euo pipefail
-
 # Test execution script for the qemu vendor image.
 # This script is supposed to run in the SDK container.
```

Comment:
```suggestion
# This script is supposed to run in the mantle container and is sourced from "test_vendor.sh".
```

Reply:
Ah, the other files have a similar statement and we can copy it instead of using my suggestion

Reply:
I updated the description.